c10/core/Allocator.h

```diff
@@ -74,7 +74,7 @@ class C10_API DataPtr {
    * However, in some cases, you know /exactly/ what the deleter
    * is, and you have a new deleter that manually wraps
    * the old one.  In this case, you can safely swap the deleter
-   * after asserting that the deleters line up.
+   * after asserting that the deleters line is up.
```

Comment:
I think "line up" in the original is intended

Reply:
Ahh yep, I see, fixed for clarity